.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+﻿
+<!--
+Add the ticket number below and uncomment
+[Link to Jira ticket](https://beisdigital.atlassian.net/browse/PC-####)
+-->
+
+# Description
+
+<!--
+Add a brief description of the change(s) you have made
+-->
+
+# Checklist
+
+- [ ] I have made any necessary updates to the documentation
+- [ ] I have checked there are no unnecessary IDE warnings in this PR
+- [ ] I have checked there are no unintentional line ending changes
+- [ ] I have added tests where applicable
+- [ ] If I have made any changes to website flow, I have checked forward and back behaviour is still consistent
+- [ ] If I have <b>added</b> a new string to the code, I have checked it is surrounded by an appropriate localiser
+- [ ] If I have <b>added</b> a new string to the code, I have added a new key/value set to the corresponding resource file
+- [ ] If I have <b>added</b> a new key to a resource file, I have checked it is not a duplicate (case insensitively)
+- [ ] If I have <b>edited</b> a resource file key, I have checked that all references have also been updated
+- [ ] If I have <b>edited</b> a resource file value, I have checked that all references to its key are still correct with the updated value.
+- [ ] If I have <b>edited</b> a string in code, I have checked that it still corresponds to a resource value key, and have created an appropriate new key and value set if not
+- [ ] If I have <b>deleted</b> a string from the code, I have checked in the corresponding resource file that it is still being referenced elsewhere, and deleted it if it is not
+- [ ] If I have interacted with strings or resource files during this ticket, I have visually confirmed that the strings/resources involved translate as expected
```

Comment:
I think this is quite a lot to digest. Could we perhaps move these to a separate markdown file and do something like:
```
[ ] If I have made any changes to content strings or resource files, I have followed the documentation
```
and link to the file?